### ISSUE TITLE
No rule to make target @VMODTOOL@

### ISSUE THREAD

**Issue by gewaechshaus** [leed25d/geoip-vmod#11]
Hey there,

while running the make we get the following error:

```
make  all-recursive
make[1]: Entering directory `/src/geoip-vmod'
Making all in src
make[2]: Entering directory `/src/geoip-vmod/src'
make[2]: *** No rule to make target `@VMODTOOL@', needed by `vcc_if.c'.  Stop.
make[2]: Leaving directory `/src/geoip-vmod/src' 
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/src/geoip-vmod'
make: *** [all] Error 2
```

How to handle this?

Best regards

**Comment by intelbg** [comment]
Friend, I had the same problem and searched a lot of time, so now I am glad to help you. You need to install the package varnish-libs-devl and the configure / make will be ok then.

**Comment by gewaechshaus** [comment]
Hey @intelbg,
thanks a l fot dude, I think i stumbled upon the same ressources...(Even if I am unsure which dev lib I finaly had installed to succed) Overall, it was a nightmare :) I had to fix a bunch of dependencies and read a lot of threads. Really hard to figure that out, but finaly I got it running since a few days :)

**Comment by intelbg** [comment]
No problem. Please tell me if now it works, because I installed it, but there are no X-GeoIP header and i doesn't work not matter that i have installed geoip and geoip devel :S

**Comment by gewaechshaus** [comment]
Yes, I've got it running... But I finaly choosed a fork from Michael Bischof you can find here:

**Comment by gewaechshaus** [comment]
Yes, I've got it running live and it's working nice so far... But I finaly choosed a fork from Michael Bischof @micbis you can find here:
https://github.com/byteworks-ch/varnish-vmod-geoip

**Comment by intelbg** [comment]
Yes, but it's not compatible with varnish 4 so it doesn't work to me. I still can't understand why header ar emissing and geoip-vmod doesn't work to me as every dependencie is installed.

**Comment by micbis** [comment]
I have updated it to be basically compatible with version 4. You can find the code for version 3 in the 3.0 branch.

**Comment by gewaechshaus** [comment]
Dank dir/ Thx a lot @micbis
